src/CommonDBTM.php

```diff
@@ -1414,6 +1408,12 @@ public function add(array $input, $options = [], $history = true)
                         );
                     }
 
+                    $this->post_addItem();
+                    if ($this instanceof CacheableListInterface) {
```

Comment:
The `post_addItem` lifecycle hook must be done after logging the `Log::HISTORY_CREATE_ITEM` action.

If we don't do that, any linked into added by `post_addItem` will be inserted before the `Log::HISTORY_CREATE_ITEM` entry.